MIGRATION.md

```diff
@@ -0,0 +1,155 @@
+| Flux CN Union       | `image/flux/controlnet` |
+| Flux CN Union Pro   | `image/flux/controlnet` |
+| Flux CN Union Pro 2 | `image/flux/controlnet` |
+
+#### Stable Diffusion Family
+
+| Display Name                       | Category                                   |
+| ---------------------------------- | ------------------------------------------ |
+| Stable Diffusion                   | `image/stable_diffusion`                   |
+| Stable Diffusion 3                 | `image/stable_diffusion_3`                 |
+| Stable Diffusion Attend and Excite | `image/stable_diffusion_attend_and_excite` |
+| Stable Diffusion DiffEdit          | `image/stable_diffusion_diffedit`          |
+
+#### aMUSEd Family
+
+| Display Name   | Category       |
+| -------------- | -------------- |
+| aMUSEd         | `image/amused` |
+| aMUSEd Img2Img | `image/amused` |
+| aMUSEd Inpaint | `image/amused` |
+
+#### Video Generation
+
+| Display Name | Category        |
+| ------------ | --------------- |
+| Allegro      | `video/allegro` |
+| Wan T2V      | `video/wan`     |
+| Wan I2V      | `video/wan`     |
+| Wan V2V      | `video/wan`     |
+| Wan VACE     | `video/wan`     |
+
+#### Audio Generation
+
+| Display Name | Category          |
+| ------------ | ----------------- |
+| AudioLDM     | `audio/audioldm`  |
+| AudioLDM 2   | `audio/audioldm2` |
+
+#### Other Pipelines
+
+| Display Name | Category          |
+| ------------ | ----------------- |
+| Würstchen    | `image/würstchen` |
+
+### Upscaling Nodes
+
+| Display Name | Replacement                                                                                                                       |
+| ------------ | --------------------------------------------------------------------------------------------------------------------------------- |
+| SPAN Upscale | Use `Diffusion Pipeline Builder` + `Generate Image (Diffusion Pipeline)` nodes<br/>[See details ↓](#for-diffusion-pipeline-nodes) |
+
+### LoRA Nodes
+
+| Display Name   | Replacement                                                                                                                 |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| Flux LoRA File | `Load LoRA` from Griptape Nodes Advanced Media Library<br/>Location: `LoRAs/Load LoRA`<br/>[See details ↓](#for-lora-nodes) |
+
+## Removed Dependencies
+
+The following Python package dependencies were removed from the Advanced Media Library:
+
+- `beautifulsoup4`
+- `protobuf` (duplicate entries consolidated)
+- `sentencepiece`
+- `spandrel`
+- `torchaudio`
+- `ftfy`
+
+## Migration Steps
+
+### For Image Processing Nodes
+
+Replace deprecated nodes with these specific nodes from the main Griptape Nodes Library:
+
+1. **Desaturate** → Replace with **`Grayscale Image`**
+
+    - Location: `Image/Edit/Grayscale Image`
+    - Same functionality: converts color images to grayscale
+
```

Comment:
The new nodes have more control over output format. You can choose PNG, JPEG, or WEBP, and you can adjust quality level.